client/src/components/user-dashboard/components/Overview.jsx

```diff
@@ -1,14 +1,21 @@
 import { Typography, TableBody, TableCell, TableHead, TableRow, Table } from '@mui/material';
+import { useState, useEffect } from 'react';
 
 const Overview = (props) => {
 
-  const dummyData = [
-    {id: 1, quiz: 'quiz 1', score: 1, lastPlayed: 'mm/dd/yyyyThh:mm:ss'},
-    {id: 2, quiz: 'quiz 2', score: 2, lastPlayed: 'mm/dd/yyyyThh:mm:ss'},
-    {id: 3, quiz: 'quiz 3', score: 3, lastPlayed: 'mm/dd/yyyyThh:mm:ss'},
-    {id: 4, quiz: 'quiz 4', score: 4, lastPlayed: 'mm/dd/yyyyThh:mm:ss'},
-    {id: 5, quiz: 'quiz 5', score: 5, lastPlayed: 'mm/dd/yyyyThh:mm:ss'}
-  ];
+  const [quizzes, setQuizzes] = useState([]);
```

Comment:
Food for thought: could probably abstract this to a custom hook if you wanted to. 

const [quizzes, getQuizzes] = useQuizzes()

Reply:
sike I saw the later commits